plugins/checks/check-deploy

```diff
@@ -53,12 +53,11 @@ checks_check_deploy() {
     local DOKKU_APP_CONTAINER_ID=${DOKKU_APP_CIDS[0]}
   fi
 
-
   # source global and in-app envs to get DOKKU_CHECKS_WAIT and any other necessary vars
   eval "$(config_export global)"
   eval "$(config_export app "$APP")"
 
-  if [[ "$(is_app_proctype_checks_skipped "$APP" "$PROC_TYPE")" == "true" ]]; then
+  if [[ "$(is_app_proctype_checks_skipped "$APP" "$DOKKU_APP_CONTAINER_TYPE")" == "true" ]]; then
```

Comment:
I *think* the variable will be available to sub-shells, and the thing that calls this trigger sets `PROC_TYPE`.